tests/Fallback/ComposerFallbackTest.php

```
@@ -49,6 +65,153 @@ public static function getSaveData(): array
+        $this->input
+            ->expects(self::any())
+            ->method('getOption')
+            ->willReturnCallback(
+                fn($option): mixed => match ($option) {
+                    $optionName => $optionValue,
+                    'verbose' => false,
+                    default => null,
+                }
+            );
+
+        $ed = $this->createMock(EventDispatcher::class);
```

PHPMD (2.15.0)

Avoid variables with short names like $ed. Configured minimum length is 3. (undefined)

(ShortVariable)